pkgs/development/ocaml-modules/linol/default.nix

```
@@ -1,37 +1,43 @@
 {
   lib,
-  fetchurl,
+  fetchFromGitHub,
   buildDunePackage,
-  yojson,
   logs,
-  lsp,
   ppx_yojson_conv_lib,
   trace,
+  uutf,
+  yojson,
 }:
 
 buildDunePackage rec {
   pname = "linol";
-  version = "0.6";
+  version = "0.10";
 
   minimalOCamlVersion = "4.14";
 
-  src = fetchurl {
-    url = "https://github.com/c-cube/linol/releases/download/v${version}/linol-${version}.tbz";
-    hash = "sha256-MwEisPJdzZN1VRnssotvExNMYOQdffS+Y2B8ZSUDVfo=";
+  src = fetchFromGitHub {
+    owner = "c-cube";
+    repo = "linol";
+    tag = "v${version}";
+    hash = "sha256-G/5nTJd+MxPgNObKW2Hmmwn4HejQ81c3f4oVXjpNSZg=";
   };
 
   propagatedBuildInputs = [
-    yojson
     logs
-    (lsp.override { version = "1.18.0"; })
     ppx_yojson_conv_lib
     trace
+    uutf
+    yojson
   ];
 
-  meta = with lib; {
+  meta = {
     description = "LSP server library";
-    license = licenses.mit;
-    maintainers = [ maintainers.ulrikstrid ];
     homepage = "https://github.com/c-cube/linol";
+    changelog = "https://github.com/c-cube/linol/releases/tag/v${version}";
```

```suggestion
    changelog = "https://raw.githubusercontent.com/c-cube/linol/refs/tags/v${version}/CHANGES.md";
```

It would be easier to handle if the changelog were in plain text format.